src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -28,7 +28,6 @@
           <div class="kuiLocalSearch">
             <input
               parse-query
-              input-focus
```

Comment:
I think ultimately if we do show this help every time the user has focus on the search bar, we should at least have a mechanism to hide it, because I can see it getting in the way, and potentially frustrating for users if they don't want to see the help any more.